_posts/2025-05-23-streamable-http-mcp.adoc

```diff
@@ -0,0 +1,107 @@
+---
+layout: post
+title: 'Quarkus MCP Server: The First Java Server to Support Streamable HTTP!'
+date: 2025-01-29
+tags: ai
+synopsis: Quarkus MCP Server is blazing a trail as the first Java MCP server with Streamable HTTP support—unlocking new possibilities for developers.
+author: maxandersen
+---
+:imagesdir: /assets/images/posts/mcpservers
+ifdef::env-github,env-browser,env-vscode[:imagesdir: ../assets/images/posts/mcpservers]
+
+The Model Context Protocol (MCP) is taking the developer world by storm, and now, with its latest spec update, it's breaking new ground: Streamable HTTP support has arrived! We're thrilled to announce that Quarkus MCP Server is the very first Java-based MCP server to embrace this innovation, making it easier than ever for you to build, experiment, and deploy MCP-powered solutions—wherever you need them.
+
+With the release of Quarkus MCP Server 1.2, Streamable HTTP joins stdio and SSE as supported transports. This means you can now connect your MCP servers to a wider range of clients and platforms, from mobile apps to cloud-native services, all with the power and flexibility of Java and Quarkus.
```

Comment:
@maxandersen It would be fair to mention that some parts are not supported yet, in particular [Resumability and Redelivery](https://modelcontextprotocol.io/specification/2025-03-26/basic/transports#resumability-and-redelivery) and the infamous and underspecified "stateless" mode.

Reply:
i've reworded it to include mention of resumablity coming later.